examples/nlp/language_modeling/bert_pretraining.py

```diff
@@ -182,12 +272,12 @@
         tokenizer = nemo_nlp.data.SentencePieceTokenizer(
             model_path=data_desc.tokenizer_model, special_tokens=special_tokens
         )
-    elif args.tokenizer == "nemo-bert":
-        logging.info("To use NemoBertTokenizer.")
-        # To train on a Chinese dataset, use NemoBertTokenizer
-        tokenizer = nemo_nlp.data.NemoBertTokenizer(pretrained_model="bert-base-uncased")
+    elif args.tokenizer == "word-piece":
+        logging.info("Using Huggingface BERT tokenizer.")
+        tokenizer = nemo_nlp.data.NemoBertTokenizer(vocab_file=data_desc.vocab_file, do_lower_case=args.do_lower_case)
```

Comment:
should WordTokenizer be initialized? also in the logging 
it doesn't have do_lower_case in params